general/app_releases/v3/v3.4.1.md

```diff
@@ -20,7 +20,7 @@ Release date: 28th February 2018
 
 ## Developers
 
-- Please refer to [https://github.com/moodlehq/moodleapp/blob/v3.4.1/upgrade.txt](https://github.com/moodlehq/moodleapp/blob/v3.4.1/upgrade.txt)
+- Please refer to [https://github.com/moodlehq/moodleapp/blob/v3.4.1/UPGRADE.md](https://github.com/moodlehq/moodleapp/blob/v3.4.1/UPGRADE.md)
```

Comment:
This link (and some others you changed) doesn't exist because the file was renamed later. Please keep the old link when needed.